packages/core-data/src/selectors.ts

```diff
@@ -257,6 +258,8 @@ export function getEntity( state: State, kind: string, name: string ): any {
 	} );
 	return getEntityConfig( state, kind, name );
 }
+getEntity.__unstableNormalizeArgs =
```

Comment:
Do we know if `__unstableNormalizeArgs` is used outside of Gutenberg? Any chance for us to break any existing normalization that relies on it? Worth double-checking in the plugins directory for example: https://wpdirectory.net/search/01K0VCNAAA2BXJKH556YBA3TCA